openfecwebapp/templates/partials/candidate/financial-summary.html

```diff
@@ -55,8 +55,8 @@ <h4 class="callout__title">
         </div>
         <div class="callout__action">
           <ul class="callout__sublinks">
-            <li><a href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle, tab='receipts') }}">Money received</a></li>
-            <li><a href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle, tab='disbursements') }}">Money spent</a></li>
+            <li><a href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle, tab='receipts') }}">Receipts</a></li>
+            <li><a href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle, tab='disbursements') }}">Disbursements</a></li>
```

Comment:
Yeah, I don't know— I need to think about this. Rather than deploying a change today, I reversed this edit, so I can consider it and come up with a well-reasoned plan.